app/services/workflow_errors_reporter.rb

```diff
@@ -32,7 +32,7 @@ def self.update_workflow(druid, process_name, error_message)
   private_class_method def self.request_params(request, druid, process_name, error_message)
     request.headers['content-type'] = "application/xml"
     request.url  "/workflow/dor/objects/druid:#{druid}/workflows/preservationAuditWF/#{process_name}"
-    request.body = "<process name='#{process_name}' status='error' errorMessage='#{error_message}'/>"
+    request.body = "<process name='#{process_name}' status='error' errorMessage='#{Rack::Utils.escape_html(error_message)}'/>"
```

Comment:
I checked the SearchWorks code, and they seem to use `CGI.escape_html` which is part of ruby stdlib:

http://ruby-doc.org/stdlib-2.5.0/libdoc/cgi/rdoc/CGI/Util.html#method-i-escape_html

https://github.com/sul-dlss/SearchWorks/blob/master/app/models/concerns/eds_subjects.rb#L43

Reply:
Cool  - ill make this fix and get the tests working! thank you